spec/integration/runner/variant-reassignment-example-17.spec.js

```diff
@@ -0,0 +1,81 @@
+ * | masterVariant: { sku: "red-bike" } | slug: { en: "bike" }                             |                    | slug: { en: "bike" }                                         |
+ * | variants: []                       | masterVariant: { sku: "red-tshirt" }             |                    | masterVariant: { sku: "red-bike" }                           |
+ * |                                    | variants: [                                      |                    | variants: []                                                 |
+ * |                                    | { sku: "green-tshirt" },{ sku: "yellow-tshirt" } |                    |                                                              |
+ * |                                    | ]                                                |                    |                                                              |
+ * +------------------------------------+--------------------------------------------------+                    +--------------------------------------------------------------+
+ * |                                    |                                                  |                    | Product:                                                     |
+ * |                                    |                                                  |                    | id: 2                                                        |
+ * |                                    |                                                  |                    | slug: { en: "bike-${timestamp}", ctsd: "${timestamp}" }      |
+ * |                                    |                                                  |                    | masterVariant: { sku: "red-tshirt" }                         |
+ * |                                    |                                                  |                    | variants: [{ sku: "green-tshirt" },{ sku: "yellow-tshirt" }] |
+ * +------------------------------------+--------------------------------------------------+--------------------+--------------------------------------------------------------+
+ */
+/* eslint-enable max-len */
+describe('Variant reassignment', () => {
+  let ctpClient
+  let product1
+  const logger = utils.createLogger(__filename)
+  const product1Sku1 = 'red-tshirt'
+  const product1Sku2 = 'green-tshirt'
+  const product1Sku3 = 'blue-tshirt'
+  const product1Slug = { en: 'bike' }
+
+  before(async () => {
+    ctpClient = await utils.createClient()
+    const products = await utils.createCtpProducts([[product1Sku1, product1Sku2, product1Sku3]],
+      ctpClient,
+      (pD) => {
+        pD.slug = product1Slug
+      })
+    product1 = products[0]
+  })
+
+  after(() =>
+    utils.deleteResourcesAll(ctpClient, logger)
+  )
+
+  it('change SKU only', async () => {
+    const reassignment = new VariantReassignment(ctpClient, logger, {}, ['brandId'])
```

Comment:
I think the `brandId` attribute and retaining attributes is not being tested here?

Reply:
https://github.com/commercetools/commercetools-node-variant-reassignment/pull/45/commits/5efb27ce7481670b7ae4fcca658521168ef37b4f